config/database.yml

```diff
@@ -79,7 +120,8 @@ test:
 #     url: <%= ENV['DATABASE_URL'] %>
 #
 production:
-  <<: *default
-  database: datacommon
-  username: datacommon
-  password: <%= ENV['DATACOMMON_DATABASE_PASSWORD'] %>
+  primary:
+    <<: *default
```

Comment:
I think we need to delete this line and also copy the pattern we used for our test database setup into the development and production keys as well to make the full array of databases available in multi-storage.

Reply:
done